example_scripts/example_runScripts_expert.py

```diff
@@ -89,7 +88,15 @@ def main(argv):
 
         # Processes software_options dictionary into Fitbenchmarking format
         minimizers, software = misc.get_minimizers(software_options)
-
+        num_runs = software_options.get('num_runs', None)
```

Comment:
Does this pattern need to be included here as well as in `fitbenchmark_group`?  Is it not dealt with there if the user doesn't define it correctly here?  If it is required then maybe it would be worth adding a comment or two explaining what it does?

Reply:
From our meeting on Friday I thought the idea was to remove the expert scirpt so this would not be a problem?

Reply:
Yes, that's fine.